magicctapipe/scripts/lst1_magic/semi_automatic_scripts/database_production/set_ra_dec.py

```diff
@@ -0,0 +1,164 @@
+"""
+Add, to LST database, infos about coordinates and extension of the sources and MC declination to be used to process the source
+
+Usage:
+$ set_ra_dec (-b YYYYMMDD -e YYYYMMDD -s source_dict -m mc_dec)
+"""
+
+import argparse
+import json
+
+import numpy as np
+import pandas as pd
+import yaml
+from astropy.coordinates import SkyCoord
+from astropy.coordinates.name_resolve import NameResolveError
+
+from magicctapipe.io import resource_file
+
+
+def main():
+
+    """
+    Main function
+    """
+
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--begin-date",
+        "-b",
+        dest="begin",
+        type=int,
+        default=0,
+        help="First date to update database (YYYYMMDD)",
+    )
+    parser.add_argument(
+        "--end-date",
+        "-e",
+        dest="end",
+        type=int,
+        default=0,
+        help="End date to update database (YYYYMMDD)",
+    )
+    parser.add_argument(
+        "--dict-source",
+        "-s",
+        dest="source",
+        type=str,
+        default="./source_dict.json",
+        help="File with dictionary of info (RA/Dec/point-source) for sources",
+    )
+    parser.add_argument(
+        "--mc-dec",
+        "-m",
+        dest="dec_mc",
+        type=str,
+        default="./dec_mc.json",
+        help="File with list of MC declinations",
+    )
+
+    args = parser.parse_args()
+    config_file = resource_file("database_config.yaml")
+
+    with open(
+        config_file, "rb"
+    ) as fc:  # "rb" mode opens the file in binary format for reading
+        config_dict = yaml.safe_load(fc)
+
+    LST_h5 = config_dict["database_paths"]["LST"]
+    LST_key = config_dict["database_keys"]["LST"]
+
+    df_LST = pd.read_hdf(
+        LST_h5,
+        key=LST_key,
+    )
+    if "ra" not in df_LST:
+        df_LST["ra"] = np.nan
+    if "dec" not in df_LST:
+        df_LST["dec"] = np.nan
+    if "MC_dec" not in df_LST:
+        df_LST["MC_dec"] = np.nan
+    if "point_source" not in df_LST:
+        df_LST["point_source"] = np.nan
+    df_LST_full = df_LST.copy(deep=True)
+    if args.begin != 0:
+        df_LST = df_LST[df_LST["DATE"].astype(int) >= args.begin]
+    if args.end != 0:
+        df_LST = df_LST[df_LST["DATE"].astype(int) <= args.end]
+
+    sources = np.unique(df_LST["source"])
+    with open(args.source) as f:
+        dict_source = f.read()
+    with open(args.dec_mc) as f:
+        mc_dec = f.read()
+    source_dict = json.loads(dict_source)
+    dec_mc = np.asarray(json.loads(mc_dec)).astype(np.float64)
+    print("MC declinations: \t", dec_mc)
+    print("\n\nChecking RA/Dec...\n\n")
+    i = 0
+    for src in sources:
+
+        try:
+            coord = SkyCoord.from_name(src)
+            if src == "Crab":
```

Comment:
I think we have a bunch of such "renamed" sources, we could put those in some dictionary